src/Resources/app/storefront/src/scss/range-slider.scss

```
@@ -0,0 +1,9 @@
+.fl--range-slider {
+  min-height: 35px;
+  padding: 2px 18px;
+  margin: -5px 12px -2px 12px;
```

The margin looks kinda odd and a bit out of sync with the other margins. Applying the following styles makes this look fine again:

```suggestion
  margin: 0 12px 5px 12px;
```